govcd/extension_test.go

```diff
@@ -28,3 +30,82 @@ func (vcd *TestVCD) Test_GetExternalNetwork(check *C) {
 	check.Assert(externalNetwork.Name, Equals, networkName)
 	check.Assert(externalNetwork.Type, Equals, expectedType)
 }
+
+func (vcd *TestVCD) Test_CreateExternalNetwork(check *C) {
+	if vcd.skipAdminTests {
+		check.Skip("Configuration org != 'Sysyem'")
+	}
+	networkName := vcd.config.VCD.ExternalNetwork
```

Comment:
Why we need configured network name for test which test creation of ext network?

Reply:
Ahh right, I forgot all the names for new resources are in `govdc/api_vcd_test.go` :+1: